building-apk.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: Building an APK for Android
+permalink: /building-apk/
+---
+
+_Note: An `apk/AndroidManifest.xml` file was already created for
+your app by `flutter init`. You can check it out if you like,
```

Comment:
Be aware of https://github.com/flutter/flutter/issues/1006.  We have other bugs on file about how this shouldn't just be for apks and how we should include an icon in the template for both ios and android.